### ISSUE TITLE
[BUG] Gesture navigation gets locked in one orientation until a launcher restart happens [A10]

### ISSUE THREAD

**Issue by NekoMimiOfficial** [Deletescape-Media/Lawnchair#4642]
### Describe the bug

After further testing my previous statement which includes that gesture nav doesn't work in landscape was false and it appears that gesture nav works fine in both orientations however it seems that its fixed to the orientation of when the launcher starts so if you change the orientation and restart the launcher it will work in the new orientation but you'll still have to restart it if you change the orientation again

It probably needs an event listener on when the orientation changes to adjust the recents screen

### Steps to reproduce

1. Start launchair
2. Switch orientation 
3. Notice the recents screen won't open
4. Restart launchair while staying at the new orientation
5. It now works but you'll have to do the same everytime you change orientation 

### Expected behavior

Works without restart

### Device information

- Device: Samsung Galaxy J6 (j6lte)
- OS: stock ROM Android version 10 (Samsung One UI 2)
- App version: 14.0.0-20240807 (goooler fork)

**Comment by NekoMimiOfficial** [comment]
This is still an issue in the latest build (pre 15)

**Comment by 0xAbdiiii** [comment]
Going from landscape to portrait mode makes the bottom swipe gesture unusable while the side gestures still work. Quick fix for that is making your device go into landscape mode and then portrait again. It gets it working for me at least.

**Comment by NekoMimiOfficial** [comment]
That's not entirely true, gesture navigations won't respond if I change the orientation until I revert back to the previous one
If my phone was in portrait and I switched to landscape gesture navigation won't work no matter how many times I switch the orientation unless I either restart lawnchair or go back to portrait, additionally you can notice the recents screen not switching orientations sometimes if you open it then switch the orientation which will keep it at the old one, seems to be an issue with the recents activity not updating it's orientation

**Comment by 0xAbdiiii** [comment]
> That's not entirely true, gesture navigations won't respond if I change the orientation until I revert back to the previous one
> If my phone was in portrait and I switched to landscape gesture navigation won't work no matter how many times I switch the orientation unless I either restart lawnchair or go back to portrait,


Are you using Quickswitch? I don't have this issue on my side at all. I change my device's orientation like 20 times per day and i have not had this happen to me once. The only noticeable thing on my side that happens is bottom swipe gesture not working sometimes.


> additionally you can notice the recents screen not switching orientations sometimes if you open it then switch the orientation which will keep it at the old one, seems to be an issue with the recents activity not updating it's orientation


True true, i just checked and it does happen but I've never encountered it on my own. Might be because i hate recents screen in landscape mode so i just subconsciously switch to portrait before i open my recents.

**Comment by NekoMimiOfficial** [comment]
Yes I'm using quick switch 3.3.8 as version 4 doesn't work for me, what version of QS are you using?

**Comment by 0xAbdiiii** [comment]
> Yes I'm using quick switch 3.3.8 as version 4 doesn't work for me, what version of QS are you using?

4.0.3 is the module version and the app version is 3.3.1

**Comment by 0xAbdiiii** [comment]
I'm on Android 12 stock rom (GO edition if that matters) using Lawnchair 14 Beta 3.

**Comment by NekoMimiOfficial** [comment]
I updated both module and app to your versions, reset the module then selected laenchair as the recents provider  after a reboot it's exactly the same, nothing changed unfortunately

**Comment by 0xAbdiiii** [comment]
> I updated both module and app to your versions, reset the module then selected laenchair as the recents provider  after a reboot it's exactly the same, nothing changed unfortunately


Well, i don't know anymore. This is weird. Could it be an OS version issue? I'm not a developer so i don't know anymore.

**Comment by ekrem-qb** [comment]
Similar issue on CrDroid Android 11

**Comment by SatoruHoshinowa** [comment]
Similar issue, but for me it only works in portrait orientation, nothing comes out if I open an application with both itself and lawnchair in landscape orientation and swipe from bottom. Running Lawnchair 14 beta 3 with QuickSwitch v4.0.3 on DotOS5.2 Android 11.

**Comment by SatoruHoshinowa** [comment]
> Similar issue, but for me it only works in portrait orientation, nothing comes out if I open an application with both itself and lawnchair in landscape orientation and swipe from bottom. Running Lawnchair 14 beta 3 with QuickSwitch v4.0.3 on DotOS5.2 Android 11.

Also I found the icons not working if i click them in a new orientation, restarting lawnchair could solve that.
There's another bug, I can't close the android settings application in recents screen, I tried to swipe it but it keeps going back, but clear all button works fine.
